docs/designers-developers/developers/tutorials/sidebar-tutorial/plugin-sidebar-1-up-and-running.md

```diff
@@ -0,0 +1,46 @@
+# Get a sidebar up and running
+
+First, we need to tell the editor that we are registering a new plugin that will have its own sidebar. We can do so by using the [`wp.plugins.registerPlugin`](https://wordpress.org/gutenberg/handbook/designers-developers/developers/packages/packages-plugins/) and [`wp.editPost.PluginSidebar`](https://wordpress.org/gutenberg/handbook/designers-developers/developers/packages/packages-edit-post/#pluginsidebar) utilities.
+
+```js
+( function( wp ) {
+	wp.plugins.registerPlugin( 'my-plugin-sidebar', {
+		render: function(){
+			return wp.editPost.PluginSidebar( {
```

Comment:
I think you should be using `wp.element.createElement` in examples. At least it is how JSX would work:
```
return wp.element.createElement(
    wp.editPost.PluginSidebar,
    props,
    'Meta field'
);
```

Reply:
f53b7e2